Readme.md

```diff
@@ -29,15 +33,6 @@ This challenge should not take more than 2-3 hours. You now have the base requir
 - [MapStore](/src/stores/MapStore.ts) might be getting a bit bloated. Time for refactoring?
 - <insert your awesome idea here!>
 
-## Backend Developers
```

Comment:
This section was added last month, but contained redundant information with the information in Notes above. I consolidated it all into a single section.